compiler/commands.nim

```diff
@@ -597,9 +597,6 @@ proc processSwitch*(switch, arg: string, pass: TCmdLinePass, info: TLineInfo;
   of "listcmd":
     expectNoArg(conf, switch, arg, pass, info)
     incl(conf.globalOptions, optListCmd)
-  of "genmapping":
-    expectNoArg(conf, switch, arg, pass, info)
```

Comment:
Make it produce a warning "genmapping is obsolete and was ignored" or something like that.